src/services/CertificateGenerationService.ts

```diff
@@ -162,6 +162,16 @@ class CertificateGenerationService {
    * @param testResult - source test result for certificate generation
    */
   public async generatePayload(testResult: any) {
+    let name = testResult.testerName;
+
+    const nameArrayList: string[] = name.split(",");
+
+    if (nameArrayList.length === 2) {
+      [nameArrayList[0], nameArrayList[1]] = [nameArrayList[1], nameArrayList[0]];
```

Comment:
I will leave it to you to decide but we could further simplify the code and use reduce some strings 'are' also arrays in Js.
See reduce https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce

```js
// I wouldn't necessarily mutate the payload
// Will flip lastName and firstName assuming the separator is the , between them
let name = testResult.testerName.split(",").reduce((curr, acc) => `${acc.trim()} ${curr}`)
...
testResult.testerName = name
...
```
One point which has not been clarified when asked in the ticket is whether the name formats can have middle names, etc..
e.g. can we have lastName, firstName, MiddleName, etc.. ?

Also even if the coverage is about what is agreed with DVSA Standard, I suggest we try to aim for 100% and add coverage for this function.
We could import it from an util.js file where this function would be added and we add the coverage for the util.js (and Name switcher function) and also checked that the function is called in generatePayload function